plugin/src/test/java/com/epam/grandhackathon/deployment/sphere/plugin/metadata/persistence/dao/DeploymentMetaDataDaoMock.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.grandhackathon.deployment.sphere.plugin.metadata.persistence.dao;
+
+import java.util.List;
+
+import com.epam.grandhackathon.deployment.sphere.plugin.metadata.model.DeploymentMetaData;
+
+public class DeploymentMetaDataDaoMock extends DeploymentMetaDataDao {
```

Comment:
OK with final, but for regular classes or interfaces we do not need this unless there is some specific use case, according to implementation I cannot see it here